packages/contracts-bedrock/deploy-config/goerli.json

```diff
@@ -1,36 +1,45 @@
 {
-  "finalSystemOwner": "DUMMY",
-  "controller": "DUMMY",
+  "numDeployConfirmations": 1,
+
+  "finalSystemOwner": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
+  "controller": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
+
+  "l1StartingBlockTag": "FILL_IN_DAY_OF",
 
-  "l1StartingBlockTag": "DUMMY",
   "l1ChainID": 5,
   "l2ChainID": 420,
   "l2BlockTime": 2,
 
-  "maxSequencerDrift": 1200,
+  "maxSequencerDrift": 600,
   "sequencerWindowSize": 3600,
-  "channelTimeout": 120,
+  "channelTimeout": 300,
 
-  "p2pSequencerAddress": "DUMMY",
+  "p2pSequencerAddress": "0x715b7219D986641DF9eFd9C7Ef01218D528e19ec",
   "batchInboxAddress": "0xff00000000000000000000000000000000000420",
-  "batchSenderAddress": "DUMMY",
+  "batchSenderAddress": "0x7431310e026B69BFC676C0013E12A1A11411EEc9",
+
   "l2OutputOracleSubmissionInterval": 20,
-  "l2OutputOracleStartingTimestamp": -1,
-  "l2OutputOracleProposer": "DUMMY",
-  "l2OutputOracleChallenger": "DUMMY",
-  "finalizationPeriodSeconds": 2,
+  "l2OutputOracleStartingBlockNumber": "FILL_IN_DAY_OF",
+  "l2OutputOracleStartingTimestamp": "FILL_IN_DAY_OF",
 
-  "l2GenesisBlockGasLimit": "0x17D7840",
-  "l2GenesisBlockBaseFeePerGas": "0x3b9aca00",
+  "l2OutputOracleProposer": "0x02b1786A85Ec3f71fBbBa46507780dB7cF9014f6",
+  "l2OutputOracleChallenger": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
 
-  "l2CrossDomainMessengerOwner": "DUMMY",
+  "finalizationPeriodSeconds": 10,
```

Comment:
Last time I checked with @trianglesphere this value was going to be discussed with @tynes @mslipper @smartcontracts . 10 seconds seems extremely low if we want to test the two-step withdrawal monitoring things. Josh/me were thinking of e.g. setting it to 10 min. (And mainnet would ofc. be a week)

Reply:
I'm fine raising it. @smartcontracts suggested 60 seconds to allow people at hackathons to easily withdraw. @K-Ho can you chime in?

Reply:
I'd strongly recommend 10 seconds - we should be optimizing here for ease of developers testing withdrawals. I assume there's plenty of ways we can test two step withdrawal monitoring even with a 10 sec finalization period, e.g. monitor every withdrawal that has been proven, but not finalized, regardless of whether or not the 10 second finalization period has elapsed.

We've had developers complain in the past about it being slow to test if they are doing withdrawals correctly given the current time required to test a single end-to-end withdrawal, and 10 minutes would be significantly worse than the current time to withdraw on Optimism Goerli (~3 min)

Reply:
I agree with @K-Ho - we should be optimizing for developers, ie: it should be possible to do e2e forked network tests on goerli using foundry (once we have the optimism state transition function in forge)

Out 2 step withdrawal infra should be able to handle the 10s withdrawal window, its not like it needs to do anything particularly complex

Reply:
12s + 12 L2 blocks is my proposal

Reply:
= 36 seconds total

Reply:
= 3 L1 blocks